### DISCUSSION TITLE
Anyone can give me the example app managing ENV via docker, please?

### DISCUSSION THREAD

**Discussion by bombxdev** [quasarframework/quasar#15093] [General - Components / Directives / etc]
According to https://quasar.dev/quasar-cli-webpack/handling-process-env, I still cannot understand it clearly.
Anyone have the example app managing environment such as DEV/PROD/UAT running via Docker container, please?

**Comment by MilosPaunovic** [comment]
Take a look at this, it might point you in the right direction: https://gist.github.com/MilosPaunovic/1da784b83466197196b4b0fd6448c0b1

**Reply by bombxdev** [reply]
@MilosPaunovic How to separate between 'development' and 'production' server without 'docker-compose build' in every time?

Normally, I use 'ENV NODE_ENV=production' in Dockerfile but I need to change to 'ENV NODE_ENV=development' and re-build it again. Is there any better ways?

**Reply by MilosPaunovic** [reply]
I don't think there is, but I'm no expert in using Docker.
